lib/src/Uncertainty/Algorithm/Analytical/Analytical.cxx

```diff
@@ -131,6 +146,18 @@ void Analytical::run()
   Point value(standardEvent.getImplementation()->getFunction().operator()(origin));
 
   result_.setIsStandardPointOriginInFailureSpace(event_.getOperator().compare(value[0], event_.getThreshold()));
+  
+  /* check if result is valid */
+  Point physicalSpaceDesignPoint(event_.getImplementation()->getAntecedent().getDistribution().getInverseIsoProbabilisticTransformation().operator()(standardSpaceDesignPoint));
+  
+  Point valuePhysicalSpaceDesignPoint(event_.getImplementation()->getFunction().operator()(physicalSpaceDesignPoint));
+  
+  const Scalar residual = std::abs(valuePhysicalSpaceDesignPoint[0] - event_.getThreshold()); 
+  
+  if (!(residual <= limitStateTolerance_))
+  throw Exception(HERE) << "Obtained design point is not on the limit state : its image by the limit state function is " << valuePhysicalSpaceDesignPoint[0] << ", which is incompatible with the threshold : " << event_.getThreshold() << " considering the limit state tolerance : "<< limitStateTolerance_;
```

Comment:
Citation (from [learnenglish.de](https://www.learnenglish.de/punctuation/punctuationtext.html)): "You do not put a space before a colon, but you do need a space after one".
```suggestion
  throw Exception(HERE) << "Obtained design point is not on the limit state: its image by the limit state function is " << valuePhysicalSpaceDesignPoint[0] << ", which is incompatible with the threshold: " << event_.getThreshold() << " considering the limit state tolerance: "<< limitStateTolerance_;
```